app/assets/javascripts/lib/date_picker.js

```diff
@@ -1,5 +1,6 @@
 $(document).on("turbolinks:load", function(){
   $(".jquery-datepicker").datetimepicker({
-    minDate: "0"
+    minDate: "0",
+    format: "Y-m-d G:i:s P"
```

Comment:
Perhapse for now we might just want to set the rails time zone to be pacific https://www.varvet.com/blog/working-with-time-zones-in-ruby-on-rails/